<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPage</class>
 <widget class="QWidget" name="SettingsPage">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin"><number>14</number></property>
   <property name="topMargin"><number>14</number></property>
   <property name="rightMargin"><number>14</number></property>
   <property name="bottomMargin"><number>14</number></property>

   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">
QFrame{background:#171a21;border-radius:16px;}
QLabel{color:white;}
QComboBox, QSlider {color:white;}
QPushButton{height:50px;border-radius:12px;color:white;background:#222634;font-size:15px;padding:8px;}
QPushButton:pressed{background:#2d3245;}
      </string>
     </property>
     <layout class="QVBoxLayout" name="layoutInner">
      <property name="leftMargin"><number>16</number></property>
      <property name="topMargin"><number>16</number></property>
      <property name="rightMargin"><number>16</number></property>
      <property name="bottomMargin"><number>16</number></property>
      <property name="spacing"><number>12</number></property>

      <item>
       <widget class="QLabel" name="lblTitle">
        <property name="text"><string>Settings</string></property>
        <property name="styleSheet"><string notr="true">QLabel{font-size:22px;font-weight:800;}</string></property>
       </widget>
      </item>

      <item>
       <layout class="QHBoxLayout" name="rowTheme">
        <property name="spacing"><number>10</number></property>
        <item><widget class="QLabel" name="lblTheme"><property name="text"><string>Thème</string></property><property name="styleSheet"><string notr="true">QLabel{font-size:16px;}</string></property></widget></item>
        <item><widget class="QComboBox" name="cmbTheme">
          <item><property name="text"><string>Nuit</string></property></item>
          <item><property name="text"><string>Jour</string></property></item>
        </widget></item>
       </layout>
      </item>

      <item>
       <layout class="QHBoxLayout" name="rowBrightness">
        <property name="spacing"><number>10</number></property>
        <item><widget class="QLabel" name="lblBright"><property name="text"><string>Luminosité</string></property><property name="styleSheet"><string notr="true">QLabel{font-size:16px;}</string></property></widget></item>
        <item><widget class="QSlider" name="sliderBrightness">
          <property name="orientation"><enum>Qt::Horizontal</enum></property>
          <property name="minimum"><number>0</number></property>
          <property name="maximum"><number>100</number></property>
          <property name="value"><number>60</number></property>
        </widget></item>
        <item><widget class="QLabel" name="lblBrightnessValue"><property name="text"><string>60</string></property></widget></item>
       </layout>
      </item>

      <item>
       <layout class="QHBoxLayout" name="rowHome">
        <property name="spacing"><number>10</number></property>
        <item><widget class="QLabel" name="lblHome"><property name="text"><string>Maison</string></property><property name="styleSheet"><string notr="true">QLabel{font-size:16px;}</string></property></widget></item>
        <item><widget class="QLineEdit" name="editHome">
          <property name="placeholderText"><string>Adresse de la maison</string></property>
          <property name="styleSheet"><string notr="true">QLineEdit{height:36px; border-radius:8px; background:#2a2f3a; color:white; padding:0 10px; font-size:14px; border:1px solid #3d4455;}</string></property>
        </widget></item>
       </layout>
      </item>

      <item>
       <layout class="QHBoxLayout" name="rowWork">
        <property name="spacing"><number>10</number></property>
        <item><widget class="QLabel" name="lblWork"><property name="text"><string>Travail</string></property><property name="styleSheet"><string notr="true">QLabel{font-size:16px;}</string></property></widget></item>
        <item><widget class="QLineEdit" name="editWork">
          <property name="placeholderText"><string>Adresse du travail</string></property>
          <property name="styleSheet"><string notr="true">QLineEdit{height:36px; border-radius:8px; background:#2a2f3a; color:white; padding:0 10px; font-size:14px; border:1px solid #3d4455;}</string></property>
        </widget></item>
       </layout>
      </item>

      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text"><string>Enregistrer</string></property>
       </widget>
      </item>

      <item>
       <spacer name="spacerV">
        <property name="orientation"><enum>Qt::Vertical</enum></property>
        <property name="sizeHint" stdset="0"><size><width>20</width><height>200</height></size></property>
       </spacer>
      </item>

     </layout>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
